markdown/en/overview-template-engine.md

```diff
@@ -4,7 +4,7 @@ FACT-Finder.
 You can take a look at all available bindings by opening your browsers **Dev Tools** (__Press F12
 Key__) and navigating to the **Network** tab. Filtering by **XHR** makes things easier.
  
-Now invoke the search, two xhr requests will appear. The second one is the one we are looking for. Click the request url,
+Now invoke the search, two xhr requests will appear. The second one is the one we are looking for. Click the request URL,
```

Comment:
In case this file is still needed, `xhr` could become `XHR`.

Reply:
also XHR requests is doppelt gemoppelt (XMLHTTPRequest requests) :)

Reply:
True, true